nextflow_schema.json

```diff
@@ -152,22 +153,22 @@
                 },
                 "skip_dotplot_o2m": {
                     "type": "boolean",
-                    "description": "Do not generate the one-to-many alignment dot-plot.",
+                    "description": "This option tells the pipeline to not generate the one-to-many alignment dot-plot.",
```

Comment:
I would prefer to keep the description short. Other descriptions do not start with _"This option tells the pipeline to"_. Can you leave that change out or suggest something more specific and concise?